python/ray/dag/compiled_dag_node.py

```diff
@@ -2355,23 +2361,45 @@ def visualize(
                 if dag_node.type_hint
                 else "UnknownType"
             ) + "\n"
-            get_output_channel_type = (
-                lambda output_channel: (
-                    type(self._channel_dict[output_channel]).__name__
-                    if output_channel in self._channel_dict
-                    else type(output_channel).__name__
-                )
-                if channel_details
-                else ""
-            )
+
+            def get_output_channel_info(output_channel, downstream_actor_id):
```

Comment:
Can we display details of `BufferedSharedMemoryChannel` as well?

Reply:
so buffered shared memory channel doesn't have any unique identifier, i can add one to it the same way intraprocess does though

Reply:
No, I meant showing the internal shared memory channels of `BufferedSharedMemoryChannel`. This might become too verbose to display and we might need another verbose level.